extensions/amp-bind/0.1/amp-state.js

```diff
@@ -43,7 +43,8 @@ export class AmpState extends AMP.BaseElement {
   activate(invocation) {
     const event = invocation.event;
     if (event && event.detail) {
-      this.updateState_(event.detail.response, /* isInit */ false);
+      this.updateState_(event.detail.response,
```

Comment:
So what will `event.detail` be here? I guess my question is: can new state get into bind's scope through this action?